ambry-tools/src/main/java/com.github.ambry/store/ConsistencyCheckerTool.java

```diff
@@ -172,31 +174,54 @@ public ConsistencyCheckerTool(ClusterMap clusterMap, StoreKeyFactory storeKeyFac
    * @return {@code true} if no real inconsistent blobs. {@code false}
```

Comment:
`@return` needs fixing. Any changes needed to the description?

Reply:
Changed